docs/_scripts/copy_basic.sh

```diff
@@ -11,6 +11,8 @@ export REPOVERSION_HREF_WITH_PKG
 http POST "$BASE_ADDR""$REPO_HREF"'modify'/ remove_content_units:="[\"$PACKAGE_HREF\"]"
 
 # Clone a repository (can be composed with addition or removal of units)
+# This operation will copy all units specified in the base_version by replacing
+# the latest available. All content that was in the repo previously wild be removed.
```

Comment:
I'm not really sure what you mean by "replacing the latest available", it's just replacing everything with the contents of the version specified.

Reply:
@ipanova This is my only objection, everything else looks great